src/Components/Books.js

```diff
@@ -1,12 +1,18 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import Bookitems from './Bookitem';
 import Input from './Form';
 
-const Books = () => (
-  <div>
-    <Bookitems title="Hunger Game" author="Suzanne Collins" />
-    <Input />
-  </div>
-);
+const Books = () => {
+  const books = useSelector((state) => state.books);
+
+  return (
+    <div className="the-container">
+      {books.map((bk) => <Bookitems key={bk.id} title={bk.title} author={bk.author} />)}
+
+      <Input />
+    </div>
```

Comment:
- [ ] Good job working on this milestone! 💯 However, as the previous reviewer suggested, please make sure you're able to delete books. At the moment I'm getting the following error in console:

![image](https://user-images.githubusercontent.com/77069011/218210007-aad98481-1005-4745-ba74-a0f869cc7cfe.png)

It looks like you missed passing the prop `id` to each BookItem. Please do so! 🤖 🤓 


```suggestion
    <div className="the-container">
      {books.map((bk) => <Bookitems key={bk.id} id={bk.id} title={bk.title} author={bk.author} />)}

      <Input />
    </div>
```